src/common/models/sources/sources.ts

```diff
@@ -16,7 +16,14 @@
 
 import { NamedArray } from "immutable-class";
 import { isTruthy } from "../../utils/general/general";
-import { Cluster, ClusterJS } from "../cluster/cluster";
+import {
+  ClientCluster,
+  Cluster,
+  ClusterJS,
+  fromConfig as clusterFromConfig,
+  serialize as clusterSerialize,
```

Comment:
```suggestion
  serialize as serializeCluster,
```
As I understand it is a function, so let's call it as a function with a pattern "do - something". I am a big fan of writing our code similar to how we speak the language, it is more natural this way.

Reply:
But I would lost my nice symmetry with `clusterFromConfig`! But you're right ... I guess.